obal/data/modules/srpm.py

```diff
@@ -0,0 +1,122 @@
+"""
+Build SRPM
+"""
+
+import shutil
+import os
+import subprocess
+from tempfile import mkdtemp
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.chdir import chdir # pylint:disable=import-error,no-name-in-module
+
+def run_command(command):
+    """
+    Run a system command
+    """
+    return subprocess.check_output(
+        ' '.join(command),
+        universal_newlines=True,
+        shell=True,
+        stderr=subprocess.STDOUT
+    )
+
+
+def copy_sources(spec_file, package_dir, sources_dir):
+    """
+    Copy RPM sources to rpmbuild environment
+    """
+    command = ["spectool", "--list-files", spec_file]
```

Comment:
not for now, but we have similar calls in `obal/data/playbooks/update/update.yaml` and `obal/data/roles/setup_sources/tasks/specfile.yml`, so it might be useful to extract the listing logic into `module_utils` and use it here and in an own module.

Reply:
#264